bin/yarn

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
```

Comment:
We are not using yarn, I'm not sure if we even need this file at all, as all it does is install packages.